### PR TITLE
[IMP] Prevent SQL Injections

### PR DESCRIPTION
**Pull request by arbaes**
Using python formatting to build SQL instructions is very insecure.
Also added .db files in `.gitignore`

### CODE REVIEW

**Comment by eibex** [comment]
Thanks! Apologies for not being aware of this.

**Comment by arbaes** [comment]
My bad too, I should have told you sooner, but you're too fast for me :D 

Also, small fix is coming

**Comment by eibex** [comment]
Sorry, too much time on my hands due to COVID-19 :P

File: rldb.py [hunk 1]
```diff
@@ -112,18 +112,18 @@ def end_creation(user, channel, message_id):
 
 
 def exists(message_id):
-    db.execute(f"SELECT * FROM messages WHERE message_id = '{message_id}';")
+    db.execute("SELECT * FROM messages WHERE message_id = ?;", (message_id,))
```
Comment (eibex) [inline rldb.py:115]: Do the commas after, for example, `message_id` serve a purpose or can I delete them? I do not see examples in the sqlite3 library using trailing commas.

Comment (arbaes) [reply rldb.py:115]: Yes, it's for forcing tuples

File: rldb.py [hunk 2]
```diff
@@ -34,7 +34,7 @@ def _generate_reactionrole_id(self):
         while True:
```
Comment (arbaes) [inline rldb.py:34]: By the way, this is not the proper way to proceed, that why we use [Primary Keys](https://www.sqlite.org/lang_createtable.html#rowid)

**Comment by arbaes** [comment]
I think I will not have time to do another PR soon, so here's a more few remarks if you have time.

- I noticed you never consider that your transactions/connections could fail, you can use try-catch with sqlite3.Error
- It's a better practice to close your connection,cursor when working with DB. I read in this case it's not a big problem as far as a single process is using the connection. But letting a cursor open could lead into memory leaks.
- There should be also some sort of conventions while working with a db. In your code, `database` is actually a connection, while `db` is actually the database cursor.

I suggest you to read [This guide](https://pynative.com/python-sqlite/), it will explain better than me :)

**Comment by eibex** [comment]
Thanks, I will study it this week.

I barely ever worked with SQL queries beforehand, sorry!

**Comment by arbaes** [comment]
No need for excuses, I just wanted to give you some insights :)

**Pull request merged**